test/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <TargetFramework Condition="$(Configuration.EndsWith('-netfx'))">net461</TargetFramework>
     <TargetFramework Condition="$(Configuration.EndsWith('-netcoreapp3_1'))">netcoreapp3.1</TargetFramework>
+    <TargetFramework Condition="'$(TestTargetFramework)' != ''">$(TestTargetFramework)</TargetFramework>
```

Comment:
>TargetFramework [](http://example.com/codeflow?start=5&length=15)

we ovrriding this value here. Is this can cause any issue? I mean can the configuration be net461 and the test netcireapp3.1 for instnce and we expect the test will run fine?

Reply:
So the issue is that the main code is `.netstandard 2.0` libraries. If I set the `TargetFramework` from the command line the build will fail because the libraries don't support targeting anything else. Currently, we get around this by making the configuration `Release-X`, and then based on what `X` is we set the target framework for the testing projects only. I was trying to find a way to set the target framework, for the tests only, to be whatever we passed in from the command line without having to add a new configuration for each one.

Do you think this is an OK way?

Reply:
I'm gonna leave it this way for now as it will unblock running tests on new frameworks without needing to add new configurations. We will want to revisit this though.